surfactant/cmd/cli.py

```diff
@@ -109,10 +123,23 @@ def add(sbom, output, output_format, input_format, **kwargs):
 
 @click.argument("sbom", type=click.File("r"), required=True)
 @click.command("edit")
-def edit(sbom, output_format, input_format, **kwargs):
+def handle_cli_edit(sbom, output_format, input_format, **kwargs):
     "CLI command to edit specific entry(s) in a supplied SBOM"
 
 
+@click.argument("outfile", type=click.File("w"), required=True)
+@click.option(
+    "--output_format",
+    is_flag=False,
+    default="surfactant.output.cytrics_writer",
```

Comment:
Use the ConfigManager here as well to check if the user has set a preferred default output format.

```suggestion
    default=ConfigManager().get(
        "core", "output_format", fallback="surfactant.output.cytrics_writer"
    ),
```